src/nuggetizer/models/nuggetizer.py

```diff
@@ -90,39 +100,55 @@ def __init__(
             self.creator_max_nuggets = creator_max_nuggets
             self.scorer_max_nuggets = scorer_max_nuggets
 
+        # Initialize logging
         self.logger = logging.getLogger(__name__)
-        self.logger.setLevel(logging.INFO if log_level > 0 else logging.WARNING)
         self.log_level = log_level
-        if self.log_level >= 1:
-            self.logger.info(
-                f"Initialized Nuggetizer with models: {creator_model}, {scorer_model}, {assigner_model}"
-            )
+        if log_level >= 1:
+            self.logger.setLevel(logging.INFO)
+        if log_level >= 2:
+            self.logger.setLevel(logging.DEBUG)
```

Comment:
what's this code doing, we don't update log_level?

Reply:
we can specify log level in the input. same debugging logs appear only  if we specify log level to be 2